frontend/store/playbackManager.ts

```diff
@@ -98,6 +98,13 @@ export const playbackManagerStore = defineStore('playbackManager', {
       queue.push(...translatedItem);
       this.queue = queue;
     },
+    async removeFromQueue(item: BaseItemDto) {
+      const queue = Array.from(this.queue);
+      const translatedItem = await this.translateItemsForPlayback(item);
+
+      queue.splice(queue.indexOf(translatedItem[0]), 1);
```

Comment:
Why fetching items from server and building up an entire queue is needed for just removing an item from the queue? Ooooops! 😅

Reply:
Yes this is a rather expensive option I used in order to get the item id. With your suggested changes this is also not relevant anymore as we take the itemid directly from the item